packages/block-library/src/video/edit.native.js

```diff
@@ -214,14 +214,14 @@ class VideoEdit extends React.Component {
 									{ showVideo && isURL( src ) &&
 										<Video
 											isSelected={ isSelected }
-											style={ videoStyle }
+											style={ { ...videoContainerStyle } }
 											source={ { uri: src } }
 											paused={ true }
 											muted={ true }
 										/>
 									}
 									{ ! showVideo &&
-										<View style={ { ...videoStyle, ...style.placeholder } }>
+										<View style={ { height: videoContainerHeight, width: '100%', ...style.placeholder } }>
```

Comment:
so the real fix here is discarding the backgroundColor prop from here right? and else is the same technically